assemblyai_haystack/transcriber.py

```diff
@@ -1,8 +1,6 @@
 from typing import Dict, List, Any, Optional
 
-from canals.serialization import default_to_dict, default_from_dict
-from haystack.preview import component, Document
-
+from haystack import component, Document, default_to_dict, default_from_dict
 from haystack.lazy_imports import LazyImport
 
 with LazyImport(message="Run 'pip install assemblyai'") as assemblyai_import:
```

Comment:
This integration will be a package, that installs `assemblyai`. So, I think there is no need to lazy import it. I expect a normal import to work.